docs/operate-and-deploy/how-it-works.md

```diff
@@ -212,6 +212,9 @@ By convention, the `ksql.service.id` property should end with a
 separator character of some form, for example a dash or underscore, as
 this makes the topic name easier to read.
 
+!!! note
+	In Confluent Cloud, the Command Topic is managed by Confluent and not visible in the Kafka cluster. Previous version of ksqlDB in Confluent Cloud created Command Topic in user-owned Kafka clusters, but they are not updated anymore.
```

Comment:
Since we haven't done the migration yet, maybe leave this as 
```suggestion
	In Confluent Cloud, the Command Topic is managed by Confluent and not visible in the Kafka cluster. Previous version of ksqlDB in Confluent Cloud created Command Topic in user-owned Kafka clusters, but this isn't the case for newly created ksqlDB clusters.
```
and then add the bit about them not being updated once we do the migration?